android/src/lib.rs

```diff
@@ -0,0 +1,101 @@
+#[macro_use]
+extern crate log;
+extern crate android_logger;
+extern crate jni;
+use self::jni::JNIEnv;
+use android_logger::Config;
+use firezone_connlib::Session;
+use jni::objects::{JClass, JObject, JString, JValue};
+use log::LevelFilter;
+
+/// This should be called once after the library is loaded by the system.
+#[allow(non_snake_case)]
+#[no_mangle]
+pub extern "system" fn Java_dev_firezone_connlib_Logger_init(_: JNIEnv, _: JClass) {
+    android_logger::init_once(
+        Config::default()
+            // Allow all log levels
+            .with_max_level(LevelFilter::Trace)
+            .with_tag("connlib"),
+    )
+}
```

Comment:
Just to not forget to do something similar later:
```suggestion
pub extern "system" fn Java_dev_firezone_connlib_Logger_init(_: JNIEnv, _: JClass) {
    #[cfg(debug_assertions)]
    let level = LevelFilter::Trace;
    #[cfg(not(debug_assertions))]
    let level = LevelFilter::Warn;
    
    android_logger::init_once(
        Config::default()
            // Allow all log levels
            .with_max_level(level)
            .with_tag("connlib"),
    )
}
```